src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -211,9 +208,13 @@
             @NotNull
             @PathVariable(value = "crn") final String crn) {
 
-        return offenderService.offenderIdOfCrn(crn)
+        return getOffenderLatestRecall(offenderService.offenderIdOfCrn(crn));
+    }
+
+    private ResponseEntity<OffenderLatestRecall> getOffenderLatestRecall(Optional<Long> maybeOffenderId) {
```

Comment:
Could convert this return `OffenderLatestRecall ` rather than `ResponseEntity<OffenderLatestRecall>`